src/main/webapp/assets/js/util/brooklyn-utils.js

```diff
@@ -175,17 +175,22 @@ define([
     };
 
     Util.logout = function logout() {
-        $.ajax({
-            type: "POST",
-            dataType: "text",
-            url: "/v1/logout",
-            success: function() {
-                window.location.replace("/");
-            },
-            failure: function() {
-                window.location.replace("/");
-            }
-        });
+        var ua = window.navigator.userAgent;
+        if (ua.indexOf("MSIE ") > 0 || ua.indexOf(" Edge/") > 0 || ua.indexOf(" Trident/") > 0) {
+            $.ajax({
+                async: false,
```

Comment:
I'd avoid this at all costs. If the server is stalled or unreachable for a short while the whole UI will freeze. See other comment for doing away with this call altogether.